rerun_cpp/src/rerun/archetypes/bar_chart_ext.cpp

```diff
@@ -1,12 +1,9 @@
 #include "../half.hpp"
 #include "bar_chart.hpp"
 
-// #define EDIT_EXTENSION
+namespace rerun::archetypes {
```

Comment:
can this be added to the clang-format file?

Reply:
Only found https://clang.llvm.org/docs/ClangFormatStyleOptions.html#compactnamespaces which isn't quite the same

But as you pointed out elsewhere there's also the option to not indent after namespace, we should try that (in a separate pr)
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#namespaceindentation